config/main.py

```diff
@@ -1392,6 +1392,27 @@ def multiasic_write_to_db(filename, load_sysinfo):
 
 def config_file_yang_validation(filename):
     config = read_json_file(filename)
+
```

Comment:
This line will also fail with empty config: https://github.com/sonic-net/sonic-utilities/blob/a87d4cc74de9a23e3a5e6da388ec4fd9dd1e6f28/config/main.py#L2081